Gemfile

```diff
@@ -44,6 +44,9 @@ group :backend do
     gem 'net-http', require: false
   end
 
+  gem 'net-smtp', require: false
+  gem 'net-imap', require: false
+  gem 'net-pop', require: false
```

Comment:
Should these be moved up together with `net-http` so they can share the guard and comment?